packages/patternfly-4/react-core/src/components/Popover/Popover.md

```diff
@@ -10,7 +10,7 @@ import { Popover, PopoverPosition, Checkbox, Button } from '@patternfly/react-co
 ## Simple popover
 ```js
 import React from 'react';
-import { Popover, PopoverPosition, Checkbox, Button } from '@patternfly/react-core';
```

Comment:
Can you change `position="right"` to `position={PopoverPosition.right}` instead?

Reply:
Not at the moment - https://github.com/patternfly/patternfly-react/issues/2099

Reply:
Oh ok, probably the PopoverPosition const needs to be an enum then? Some of the other examples still use PopoverPosition it seems

Reply:
<img width="792" alt="Screen Shot 2019-05-29 at 2 53 15 PM" src="https://user-images.githubusercontent.com/5942899/58583368-d7595b80-8221-11e9-82ad-c9e3fcd515c1.png">

Reply:
Sorry only now does GitHub finally show your comment from an hr ago.. Yep, that's the convention for finite sets of values like this.

I was only removing it from the example where it wasn't used at all, had fixed that locally before the unrelated regression of position property happened. I can remove the use of PopoverPosition object from the other examples but this type is in the process of being fixed and the functionality will be restored soon. I can remove these changes from this PR but hoping we can take this in as-is since the incremental step does improve things and also unblocks the bug we have for SelectProps. Would this be ok for a short term improvement?

Reply:
sounds good thanks @seanforyou23